lightning/src/ln/channel.rs

```diff
@@ -1214,6 +1236,10 @@ pub(super) struct ChannelContext<SP: Deref> where SP::Target: SignerProvider {
 	secp_ctx: Secp256k1<secp256k1::All>,
 	channel_value_satoshis: u64,
 
+	/// Info about an in-progress, pending splice (if any), on the pre-splice channel
+	#[cfg(splicing)]
+	pub(crate) pending_splice_pre: Option<PendingSpliceInfoPre>,
```

Comment:
Is this only relevant to funded channels? Should we move it there instead?

Reply:
You mean the ChannelPhase? I would put there only larger structs, not individual fields.
This field is also used to carry state from `splice_channel` till `channel_ack`, and during that time the channel is not actually splicing yet.

Reply:
Directly in `Channel` which is held by the `Funded` variant of `ChannelPhase`.

Reply:
I have considered this, and changed it accordingly.